mmdeploy/codebase/mmdet/deploy/object_detection.py

```diff
@@ -316,7 +316,9 @@ def get_postprocess(self, *args, **kwargs) -> Dict:
                 params['mask_thr_binary'] = params['rcnn']['mask_thr_binary']
         if 'mask_thr_binary' in params:
             type = 'ResizeInstanceMask'  # for instance-seg
-            params['is_resize_mask'] = False  # resize and crop mask default
+            # resize and crop mask to origin image
+            params['is_resize_mask'] = True
```

Comment:
Why change the default value of params['is_resize_mask'] ?

Reply:
@irexyc It should be set as True, which is consistent with old version, when masks are resized to original image shape(28x28). This is a bug brought by https://github.com/open-mmlab/mmdeploy/pull/2229